demos/src/demo.scss

```diff
@@ -50,7 +49,7 @@
 }
 
 .custom-toggle,
-.custom-toggle[aria-pressed] {
+.custom-toggle[aria-expanded="true"] {
 	background: red;
 	i:after {
 		content: ' x';
```

Comment:
I know this is a demo but an X should not be used in place of the explicit word 'close'. People might copy this so it might be a good chance to change it!